index.html

```diff
@@ -1,62 +1,94 @@
+            margin: 5;
+            cursor: pointer;
+        }
 
-    <script>
-        var calculator = {
-            addNumbers: function () {
-                // hint: these are strings
-                var numberOne = document.forms.inputs.numberOne.value;
-                var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
-            },
-            subtractNumbers: function () {
-                
-            }
+        .textview {
+            width: 240;
+            margin: 3;
+            font-size: 40;
+            padding: 3;
         }
+    </style>
+
 
-    </script>
 
 </head>
 
 <body>
-    <div>Header</div>
-    <div>
-       <!--
-         - allow two numbers to be entered
-         - show buttons for add and subtract
-         - show the result
-        -->
-        <form name="inputs">
-            <div>
-                Number 1:
-                <input type="number" name="numberOne" class="form-control" placeholder="First number" aria-label="Number1" >
-            </div>
-            <div>
-                Number 2:
-                <input type="number" class="form-control" name="numberTwo">
-            </div>
-            <div>
-                <button type="button" class="btn btn-primary" onclick="calculator.addNumbers()" >Add</button>
-                <button type="button" class="btn btn-primary">Substract</button>
-            </div>
-            <div>
-                <input name="result" value="0" class="form-control" disabled />
-            </div>
+    <div class="main">
+        <form name="form">
+            <input class="textview" name="textview">
         </form>
+        <table>
+            <tr>
+                <td><input class="button" type="button" value="1" onclick="insert(1)"></td>
+                <td><input class="button" type="button" value="2" onclick="insert(2)"></td>
+                <td><input class="button" type="button" value="3" onclick="insert(3)"></td>
+                <td><input class="button" type="button" value="+" onclick="insert('+')"></td>
+            </tr>
+            <tr>
+                <td><input class="button" type="button" value="4" onclick="insert(4)"></td>
+                <td><input class="button" type="button" value="5" onclick="insert(5)"></td>
+                <td><input class="button" type="button" value="6" onclick="insert(6)"></td>
+                <td><input class="button" type="button" value="-" onclick="insert('-')"></td>
+            </tr>
+            <tr>
+                <td><input class="button" type="button" value="7" onclick="insert(7)"></td>
+                <td><input class="button" type="button" value="8" onclick="insert(8)"></td>
+                <td><input class="button" type="button" value="9" onclick="insert(9)"></td>
+                <td><input class="button" type="button" value="*" onclick="insert('*')"></td>
+            </tr>
+            <tr>
```

Comment:
For this assignment, these are the only buttons needed for Add, Subtract, Multiply, Divide.